llvm/lib/Transforms/Vectorize/VPlanTransforms.cpp

```diff
@@ -956,6 +975,14 @@ static Value *tryToFoldLiveIns(const VPRecipeBase &R, unsigned Opcode,
   }
 
   InstSimplifyFolder Folder(DL);
+  if (OpcodeOrIID.first) {
```

Comment:
Should we just call `getOpcodeOrIntrinsicID` on R here instead of plumbing it through as an argument?

Reply:
Good idea. Will update patch after @fhahn's bugfix to CSE lands.